docs/source/resources/glossary.md

```diff
@@ -36,7 +36,7 @@ You can use Kedro-Viz to visualise your Kedro data pipelines:
 [Further information from the Kedro-Viz repository](https://github.com/kedro-org/kedro-viz) and [Kedro-Viz documentation](../visualisation/kedro-viz_visualisation.md)
 
 ## Layers (data engineering convention)
-According to [data engineering convention](../faq/faq.md#what-is-data-engineering-convention), a pipeline can be broken up into different layers according to how data is processed. This convention makes it easier to collaborate with other team members because everyone has an idea of what type of data cleaning or processing has happened.
+According to [data engineering convention](https://towardsdatascience.com/the-importance-of-layered-thinking-in-data-engineering-a09f685edc71), a pipeline can be broken up into different layers according to how data is processed. This convention makes it easier to collaborate with other team members because everyone has an idea of what type of data cleaning or processing has happened.
```

Comment:
Also, for this glossary, maybe we want to use Sphinx semantic tags? https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-term (not for this PR, and especially not if we end up deciding against Sphinx)